src/python/grpcio/grpc/_cython/_cygrpc/aio/callback_common.pyx.pxi

```diff
@@ -128,7 +129,7 @@ async def _receive_message(GrpcCallWrapper grpc_call_wrapper,
         # the callback (e.g. cancelled).
         #
         # Since they all indicates finish, they are better be merged.
-        _LOGGER.debug(e)
+        _LOGGER.debug('Failed to receive any message from Core')
```

Comment:
why not `_LOGGER.exception(....)`?

Reply:
Receiving message operation can fail under normal RPC procedure. For example, the client sends `done_writing` or cancels. To me, it's a benign error that we should notify user without alarming them.